Services/Taskbar/TaskbarButton.cpp

```diff
@@ -60,7 +60,7 @@ void TaskbarButton::resize_event(GUI::ResizeEvent& event)
 static void paint_custom_progress_bar(GUI::Painter& painter, const Gfx::IntRect& rect, const Gfx::IntRect& text_rect, const Palette& palette, int min, int max, int value, const StringView& text, const Gfx::Font& font, Gfx::TextAlignment text_alignment)
 {
     float range_size = max - min;
-    float progress = (value - min) / range_size;
+    float progress = (value - min) / (range_size > 0 ? range_size : 1);
```

Comment:
Please use `0.0` and `1.0`, otherwise it can be interpreted as an integer. Also adding a `f` as a suffix doesn't hurt either

Reply:
on it, thanks for catching that.